src/languageservice/utils/strings.ts

```diff
@@ -75,3 +75,16 @@ export function safeCreateUnicodeRegExp(pattern: string): RegExp {
     return new RegExp(pattern);
   }
 }
+
+export function getLastWhitespaceAfterChar(str: string, offset: number): number {
```

Comment:
Is this a good name for what this function does. It is not really covering all whitespace characters.